validate_sample.py

```diff
@@ -43,7 +43,7 @@ def validate_header_row(fieldnames):
                     'HTC_DIGITAL', 'FIELDCOORDINATOR_ID', 'FIELDOFFICER_ID', 'TREATMENT_CODE',
                     'CE_EXPECTED_CAPACITY']
 
-    if not set(fieldnames).issubset(valid_header):
+    if not set(fieldnames) == set(valid_header):
```

Comment:
Header could be a set literal instead of a list then converted (like `{'ARID', 'ESTAB_ARID' , ...}`)